src/Controls/src/Core/DefinitionCollection.cs

```diff
@@ -101,13 +102,15 @@ public void RemoveAt(int index)
 			OnItemSizeChanged(this, EventArgs.Empty);
 		}
 
-		public event EventHandler ItemSizeChanged;
+		public event EventHandler ItemSizeChanged
+		{
+			add => _weakEventManager.AddEventHandler(value);
+			remove => _weakEventManager.RemoveEventHandler(value);
+		}
```

Comment:
There are some existing tests that check this event still fires:

https://github.com/dotnet/maui/blob/cda3eb3381cfb686567ed05e3eb8e8f26c02d785/src/Controls/tests/Core.UnitTests/Layouts/GridLayoutTests.cs#L77-L78

https://github.com/dotnet/maui/blob/cda3eb3381cfb686567ed05e3eb8e8f26c02d785/src/Controls/tests/Core.UnitTests/Layouts/GridLayoutTests.cs#L98-L99